includes/class-gdpr-audit-log.php

```diff
@@ -104,9 +104,12 @@ public static function get_log( $email, $token = null ) {
 			$file_found   = file_exists( $path . $filename );
```

Comment:
I think, that for VIP we should get the $wp_filesystem and use exists() method.

Reply:
Also, am I correct thinking that contents of `gdpr_logs/` are going to be publicly available and this is "security by obscurity"?

Reply:
So full VIP support of audit log is something I want to look at more next. I *think* this should work but would need to look into more closely.